test/jit/test_pdt.py

```diff
@@ -224,3 +224,63 @@ def test_model_with_args(a, m):
         user_class = class_with_args(False)
         scripted_fn = torch.jit._script_pdt(test_model_with_args, example_inputs=[(10, user_class, ), (10.9, user_class, ), ])
         self.assertEqual(scripted_fn(100, class_with_args(True), ), test_model_with_args(100, class_with_args(True)))
+
+    def test_nn_module(self):
+        class Foo(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, x):
+                assert x is not None
+                return x
+
+        make_global(Foo)
+        scripted_model = torch.jit._script_pdt(Foo(), example_inputs=[(100, ), (10.80, ), ])
+        input = 120
+        self.assertEqual(scripted_model(input), input)
+
+    def test_nn_module_2(self):
+        class model(torch.nn.Module):
```

Comment:
```suggestion
        class Model(torch.nn.Module):
```

Also, try to use a meaningful name for structures used in tests rather than generic names.